kelly-mundle.html

```diff
@@ -0,0 +1,47 @@
+<html>
+  <head>
+    <title>CSS Timeline</title>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta name="description" content="A Timeline of CSS"> <!--added for clarity-->
+    <link rel="stylesheet" href="/kelly-mundle.css">
+  </head>
+  <body>
+    <h1>A Brief History of CSS</h1> <!--some minor edits... ha-->
+      <ul>
+        <li><div id="list-item-1"><span class="date">1994:</span> The first work on style sheets started at CERN.</div></li>
```

Comment:
I believe I remedied it. Used span again, instead. I was taking slides too literally!